### PR TITLE
Add OpenStreetMap Places page

### PR DESCRIPTION
**Pull request by eneemann**
initial prep for OSM places page

### CODE REVIEW

**Comment by eneemann** [comment]
> 

That's the URL I use for the GET request to query the Overpass API.  I'm not quite sure how to present it.  I had to fix my HTML code, but the link now points to the Overpass interface where a user can run the query and export the results...I think I'll just change the text to show that, too.

File: data/society/openstreetmap-places/index.html [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        <li>website - the feature's website in the OSM database, if available
+        <li>phone - the feature's phone number(s) in the OSM database, if available
+        <li>open_hours - the feature's operating hours in the OSM database, if available
+        <li>osm_addr - the feature's address in the OSM database, if available
+      </ul>
+    <h4 id="DataSources">Data Sources</h4>
+    <ul>
+        <li>The original bulk set of OSM data for the state of Utah is downloaded from <a href="https://download.geofabrik.de/index.html">Geofabrik</a>:
+            <ul>
+                <li><a href="http://download.geofabrik.de/north-america/us/utah-latest-free.shp.zip">http://download.geofabrik.de/north-america/us/utah-latest-free.shp.zip</a></li>
```
Comment (steveoh) [inline data/society/openstreetmap-places/index.html:94]: **nitpick**: does this have an https url?

File: data/society/openstreetmap-places/index.html [hunk 2]
```diff
@@ -60,32 +60,32 @@ <h4 id="DataCreation">Data Creation Process</h4>
...
-        <li>lat - the approximate latitude of the feature, calculated in WGS84 [EPSG:4326] (for the most precision, it is recommended that the feature's geometry is used instead of this field)
-        <li>amenity - the amenity available at the feature (if applicable), often similar to the category
-        <li>cuisine - the type of food available (if applicable), multiple types are separated by semicolons (;)
-        <li>tourism - the type of tourist location, if applicable (zoo, viewpoint, hotel, attraction, etc.)
-        <li>shop -  the type of shop, if applicable
-        <li>website - the feature's website in the OSM database, if available
-        <li>phone - the feature's phone number(s) in the OSM database, if available
-        <li>open_hours - the feature's operating hours in the OSM database, if available
-        <li>osm_addr - the feature's address in the OSM database, if available
+        <li><code>addr_dist</code> - the distance (m) to the nearest UGRC address point within 25 m
```
Comment (steveoh) [inline data/society/openstreetmap-places/index.html:69]: **thought**: do we want to group the attributes near each other where it makes sense.

It seems addr_dist, ugrc_addr, and osm_addr would logically group together.

Comment (eneemann) [reply data/society/openstreetmap-places/index.html:69]: Yeah, I think that's a better idea

**Review by stdavis — APPROVED** [review]

**Pull request merged**